packages/swapper/src/swappers/thorchain/ThorchainSwapper.ts

```diff
@@ -145,6 +146,13 @@ export class ThorchainSwapper implements Swapper<ChainId> {
         })
         const txid = await adapter.broadcastTransaction(signedTx)
         return { tradeId: txid }
+      } else if (trade.sellAsset.chainId === KnownChainIds.CosmosMainnet) {
```

Comment:
Not needed as part of this PR as this just follows the existing if/else pattern but time to switch to a switch statement?

Reply:
switch statment isnt any better really. just stylistic opinion